packages/familie-form-elements/src/checkbox/FamilieCheckbox.tsx

```diff
@@ -11,24 +11,17 @@ export const FamilieCheckbox: React.FC<ICheckboxProps> = ({
     checked,
     className,
     erLesevisning,
-    label,
-    onChange,
```

Comment:
Gjennomgående: jeg har fjernet eksplisitt nevning av props som kun sendes videre, slik at de heller inngår i `...props` som uansett sendes videre til underliggende komponent. Props som brukes til evaluering internt i denne komponenten, eller skal brukes også for lesevisningskomponenten, nevner vi fremdeles her